### PR TITLE
Resolve issues with Apex Code Coverage

### PR DESCRIPTION
**Pull request by estebanefi**
**Requirements**

- [x] I have added test coverage for new or changed functionality
- [x] I have followed the repository's [pull request submission guidelines](../blob/main/CONTRIBUTING.md#submitting-pull-requests)
- [x] I have validated my changes against all supported platform versions

**Related issues**

#9 

**Describe the solution you've provided**

Added apex unit tests so that we can meet the minimum 75% coverage requirements for any Apex deployed to a Salesforce Org.

image

This will allow other users to use launch darkly in apex without having to write tests

**Describe alternatives you've considered**

NA

**Additional context**

NA

### CODE REVIEW

**Comment by cwaldren-ld** [comment]
Hi @estebanefi , thank you very much for this contribution!

Because the `prettier` formatting has touched the whole codebase, it makes it difficult to review the changes. Would you mind submitting another PR with just the results of `prettier` run on the codebase?

Once that is merged, we can rebase this PR so that the unit tests are the only changes.

File: force-app/test/LDConfigTest.cls-meta.xml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ApexClass xmlns="http://soap.sforce.com/2006/04/metadata">
+    <apiVersion>59.0</apiVersion>
+    <status>Active</status>
+</ApexClass>
```
Comment (cwaldren-ld) [inline force-app/test/LDConfigTest.cls-meta.xml:5]: Please add a newline here

File: force-app/test/VersionedDataTest.cls-meta.xml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ApexClass xmlns="http://soap.sforce.com/2006/04/metadata">
+    <apiVersion>59.0</apiVersion>
+    <status>Active</status>
+</ApexClass>
```
Comment (cwaldren-ld) [inline force-app/test/VersionedDataTest.cls-meta.xml:5]: newline

File: force-app/test/LDConfigTest.cls [hunk 3]
```diff
@@ -0,0 +1,73 @@
+/**
```
Comment (cwaldren-ld) [inline force-app/test/LDConfigTest.cls:1]: Is this comment needed?

Comment (estebanefi) [reply force-app/test/LDConfigTest.cls:1]: nope, removing. It is caused by auto generating the test class

File: force-app/test/VersionedDataTest.cls-meta.xml (hunk 2 above)
Comment (estebanefi) [reply force-app/test/VersionedDataTest.cls-meta.xml]: done!

File: force-app/test/LDConfigTest.cls-meta.xml (hunk 1 above)
Comment (estebanefi) [reply force-app/test/LDConfigTest.cls-meta.xml]: done!

**Comment by cwaldren-ld** [comment]
Hi @estebanefi , please let me know if you ready for this to be merged, or need to perform any other testing.

**Comment by estebanefi** [comment]
@cwaldren-ld this is good to merge, validated it all deployed to a sandbox without any issues.

**Comment by cwaldren-ld** [comment]
Thanks - this should be contained in our next release.

**Pull request merged**